dusk/grammar.py

```diff
@@ -474,8 +481,8 @@ def field_access_expr(self, field: DuskField, index: expr = None):
             raise DuskSyntaxError(
                 f"Invalid field access {name} outside of a vertical region!"
             )
-        return make_field_access_expr(
-            field.sir.name, self.field_index(index, field=field)
+        return make_unstructured_field_access_expr(
+            field.sir.name, *self.field_index(index, field=field)
```

Comment:
why is the unary `*` needed here?

Reply:
It _unpacks_ the tuple that `self.field_index` returns. Basically these are all roughly equivalent:

```python
calling_some_function(*[a, b, c])
calling_some_function(*(a, b, c))
calling_some_function(a, b, c)
some_tuple = (a, b, c)
calling_some_function(*some_tuple)
```